tests/ui/crashes/ice-7410.rs

```diff
@@ -6,6 +6,7 @@
 #![no_std]
 #![allow(clippy::if_same_then_else)]
 #![allow(clippy::redundant_pattern_matching)]
+#![allow(clippy::empty_drop)]
```

Comment:
Since this is a restriction lint now, those allows on unrelated tests shouldn't be necessary anymore.